src/scikitlearn.jl

```diff
@@ -49,8 +49,7 @@ ScikitLearnBase.predict_log_proba(gmm::GMM, X) = log(gmmposterior(gmm, X)[1])
 ScikitLearnBase.predict_proba(gmm::GMM, X) = gmmposterior(gmm, X)[1]
 ScikitLearnBase.predict(gmm::GMM, X) =
     # This is just `argmax(axis=2)`. It's very verbose in Julia.
-    ind2sub(size(X),
-            vec(findmax(ScikitLearnBase.predict_proba(gmm, X), 2)[2]))[2]
+    CartesianIndices(size(X))[vec(findmax(ScikitLearnBase.predict_proba(gmm, X), dims=2)[2])][2]
```

Comment:
I think that in 0.7/1.0, this line should just be `getindex.(argmax(ScikitLearnBase.predict_proba(gmm, X), dims=2), 2)`. Then the test might work.

Reply:
Fixed and the test works, though the intent isn't entirely clear given my quick read. Half of the return value is 1 and half is 2, and the assert now passes.

Reply:
The training set has 300 points in gaussian #1 and 600 in gaussian #2. They are cleanly separated. After performing clustering, how many points should be in cluster #1? Either 300 or 600, since we don't know which cluster corresponds to which gaussian.